packages/suite-web/e2e/tests/onboarding/t2t1-recovery-success.test.ts

```diff
@@ -7,7 +7,7 @@ describe('Onboarding - recover wallet T2T1', () => {
         cy.viewport(1080, 1440).resetDb();
         cy.prefixedVisit('/');
 
-        cy.task('startEmu', { wipe: true, version: '2.4.3' });
```

Comment:
here we should also test the latest one but I believe I was hit by this bug https://satoshilabs.slack.com/archives/CL1D61PQF/p1698389109047929
cc @trezor/qa